### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by MattKraatz**
```
git pull --prune
git checkout readme
```

Please review! Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Bangazon Workforce Management
```
Comment (matthamil) [inline README.md]: `# Customer Service Incident Report`

**Review by matthamil — CHANGES_REQUESTED** [review]
LGTM other than name fix

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Dependencies
+
+To ensure that the  Bangazon Workforce Management App works as intended make sure that you have the following dependencies and technologies on your local machine
```
Comment (matthamil) [inline README.md]: Workforce Management -> Customer Service Incident Report

**Comment by MattKraatz** [comment]
Name's fixed

**Comment by daynewright** [comment]
✍️ LGTM 👍

**Review by matthamil — APPROVED** [review]

**Pull request merged**